package.json

```diff
@@ -16,12 +16,17 @@
   "devDependencies": {
     "@babel/cli": "^7.6.4",
     "@babel/core": "^7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.7.0",
     "@babel/preset-env": "^7.6.3",
     "@babel/preset-typescript": "^7.6.0",
     "@babel/register": "^7.6.2",
+    "@babel/runtime": "^7.7.2",
```

Comment:
do we need this?

Reply:
It's needed in order to run jest tests, or at least I can't run them without `@babel/runtime`.